draft-js-video-plugin/src/video/components/DefaultVideoComponent.js

```diff
@@ -0,0 +1,49 @@
+import React, { PropTypes } from 'react';
+import utils from '../utils';
+
+const YOUTUBE_PREFIX = 'https://www.youtube.com/embed/';
+const VIMEO_PREFIX = 'https://player.vimeo.com/video/';
+
+const getSrc = ({ src }) => {
+  const {
+    isYoutube,
+    getYoutubeSrc,
+    isVimeo,
+    getVimeoSrc,
+  } = utils;
+  if (isYoutube(src)) {
+    const { srcID } = getYoutubeSrc(src);
+    return `${YOUTUBE_PREFIX}${srcID}`;
+  }
+  if (isVimeo(src)) {
+    const { srcID } = getVimeoSrc(src);
+    return `${VIMEO_PREFIX}${srcID}`;
+  }
+  return undefined;
+};
+
+const DefaultVideoCompoent = (props) => {
+  const { blockProps, style } = props;
+  const src = getSrc(blockProps);
+  if (src) {
+    return (
+      <div style={style} >
+        <div style={{ width: '100%', height: '0', position: 'relative', paddingBottom: '56.25%' }} >
```

Comment:
@mzbac could we live without these wrapping divs? I think size and positioning should be decorator. Otherwise decorators like resizeable won't work in combination with it.